app/src/main/java/com/jerboa/ui/components/common/AccountHelpers.kt

```diff
@@ -27,6 +27,7 @@ fun getCurrentAccount(accountViewModel: AccountViewModel): Account {
     return acc
 }
 
+// TODO why is this necessary when .toEnum() exists?
```

Comment:
I believe this is about `getEnumFromIntSetting` ?

Why it exists over `toEnum`. Its rather a specialized `toEnumSafe`.  If we remove an entry but someone still has its index in the DB.  It would then fallback to default which might not be the first entry.

Its rather niche as if we someone removes an entry they should also include a query to reset that index.
But it was just incase.

Reply:
I'll add a comment there about this.